draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -6897,72 +6897,94 @@ Accept-Language: da, en-gb;q=0.8, en;q=0.7
    <x:anchor-alias value="Vary"/>
 <t>
    The "Vary" header field in a response describes what parts of a request
-   message, aside from the method and target URI,
-   might influence the origin server's process for selecting and representing
-   this response.
+   message, aside from the method and target URI, might influence the
```

Comment:
```suggestion
   message, aside from the method and target URI, might have influenced the
```

Putting this in the past tense relative to the interpretation of the response makes it easier to understand what's going on.